.github/workflows/deploy-environment.yml

```diff
@@ -1,13 +1,15 @@
 name: Deploy Argilla environment
 
 on:
+  workflow_dispatch:
   workflow_call:
     inputs:
       image-name:
         description: The name of the Docker image to deploy.
         type: string
+        default: argilla/argilla-quickstart-for-dev
```

Comment:
Why did you change this? It's an image that still doesn't exists?

Reply:
I added the image name as default. This will simplify the manual deployment execution. This image is created in the [previous step](https://github.com/argilla-io/argilla/blob/ci/manual-pr-deploy-and-cleaning-services-one-by-day/.github/workflows/package.yml#L155). At this point, it will be available to use.